src/components/AddNewVisitModal.vue

```diff
@@ -298,9 +363,40 @@ export default defineComponent({
       return null
     }
   },
+  mounted() {
+    if (this.patientId) {
+      this.fetchPatientData()
+    }
+  },
   methods: {
-    // POST request to add a new patient / PUT request to update an existing patient
-    // If isAdd is true, do insert patient, otherwise do update patient
+    // Preloads existing patient data
+    async fetchPatientData() {
```

Comment:
Would it be better to pass in the Patient prop here when fetching the patient initially instead of fetching the entire patient visit, since the fields are going to be edited anyway